extensions/adobe/experience/campaign/experienceevent.schema.json

```diff
@@ -257,6 +257,11 @@
                 "Business qualifier that identifies the event sent by the data source. It informs on the business reason for sending the event. It is unique per organization. This is used by Campaign orchestration to identify the event without inspecting its payload to determine which action should be triggered when the event is received. The value of this field is a contract between Campaign orchestration and the data source."
             }
           }
+        },
+        "xdm:mutation": {
+          "title": "Mutated fields from the previous experienceEvent.",
+          "$ref": "https://ns.adobe.com/experience/campaign/experienceevent",
+          "description": "Information of the fields that have changed since the previous experienceevent to keep track of the state changes across events. For e.g. if when an email gets sent, an eE with metric `sends` will be sent. This may get bounced and an eE with metric `bounces` will be sent then. It may be important for a downstream application to know that the previous eE with metric `sends` is now to be invalidated. This field will contain the values from the previous eE that have a different value in the current eE."
```

Comment:
@saurabhere please change "eE" to "Experience Event"

Reply:
damn! I was hoping it would stick.